Cargo.toml

```diff
@@ -31,7 +31,7 @@ log      = "0.4"
 slab     = "0.4.0"
 net2     = "0.2.29"
 iovec    = "0.1.1"
-cfg-if   = "=0.1.9"
```

Comment:
was there a reason for the “=“ dep, or is this just lost to the ages?

Reply:
no idea.

Reply:
Oh, to match the min rust version  mio targets.

Reply:
I reverted to keep it simple.